docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -0,0 +1,111 @@
+= Plugin Mobile App
+
+== General Info
+
+The plugin provides functionality to test mobile application on real devides, simulators and emulators
+
+== Installation
+
+.build.gradle
+[source,gradle]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-mobile-app', version: '<version>')
+----
+
+== Properties
+
+=== Required
+
+* `selenium.grid.device-name` - defines device name to run tests on
+* `selenium.grid.username` - defines remote grid username
+* `selenium.grid.password` - defines remote grid password
+* `selenium.grid.host` - defines remote grid host
+
+=== Optional
+
+* `selenium.grid.automation-name` - defines automation engine to use, e.g. `UIAutomator2` for `Android` platform and `XCUITest` for `iOS` platform
```

Comment:
can we provided table-like view with mapping Mobile OS <> recommended automation engine